src/rgw/rgw_file.cc

```diff
@@ -1520,11 +1523,11 @@ namespace rgw {
   }
 
   int RGWWriteRequest::exec_start() {
-    struct req_state* s = get_state();
+    struct req_state* _s = get_state();
```

Comment:
I don't like naming variables _something, esp local

Reply:
ok. will rename to "state".
a better fix was to change "s" in the parent where its is "protected" to "private" - but this would probably cause too many code changes in other places...

Reply:
ok, that's a good solution